### PR TITLE
Updated with title attributes and ALT tags

### PR DESCRIPTION
**Pull request by ProfCyberNaught**
## Summary

**Updated README.md:**
Added title attribute tags to profile links
Added title attribute tags to other links
Added ALT image tags where required

**Updated CONTRIBUTING.md:**
Added title attribute tags to links

**Updated CODE_OF_CONDUCT.md:**
Added title attribute tags to links

## Type of change

- [x] Updated documentation (updated the readme, templates, or other repo files)

## How Has This Been Tested?

- [x] Checked changes made for spelling
- [x] Checked links work
- [x] Checked profile pages still using feature
- [x] Reported some profiles are not linking back to repository or demo site

## Additional Information

I physically checked all profile links to make sure the example profiles were still using this feature. Positively, all profiles still use this feature **but some are not linking back to the repository or demo site**.

The following are using but not linking back as per the source link code provided:

https://github.com/jcs090218 - Tree Profile Image
https://github.com/yousafesaeed - CRT Monitor Image
https://github.com/HectorSaldes - White Shirt Glasses Image
https://github.com/Maagnitude - Guy Beard Grey Top Image
https://github.com/quiet-node - Smiling Guy Black Jacket Image
https://github.com/wiz64 - White Cat Image
https://github.com/thakurballary - Guy Black Beard & Hair White Top Image
https://github.com/BenjaminMichaelis - Guy Smiling Red Cap
https://github.com/PROxZIMA - Guy Black Hair & Top Glasses
https://github.com/angelofallars - Guy Red Profile Image
https://github.com/tavignesh - Never Settle with Hat Image
https://github.com/Tarun-Kamboj - Black & White Eternal Image
https://github.com/8BitJonny - Dark Face Mask Glasses Image

This is a result of 13/41 meaning on average, **32% or 1/3rd of users do not actively provide a link back to the repository**. _Seems a shame to use a feature and not really give credit to the developer._

### CODE REVIEW

**Comment by ProfCyberNaught** [comment]
Should be able to close Issue #200 when merged.

File: README.md [hunk 1]
```diff
@@ -33,55 +33,55 @@ Here you can easily customize your Typing SVG with a live preview.
...
+[![majidtdeni666](https://github.com/majidtdeni666.png?size=60)](https://github.com/majidtdeni666 "majidtdeni666 on GitHub")
+[![GalexY727](https://github.com/galexy727.png?size=60)](https://github.com/galexy727 "GalexY727 on GitHub")
+[![HectorSaldes](https://github.com/HectorSaldes.png?size=60)](https://github.com/HectorSaldes "HectorSaldes on GitHub")
+[![Ash-codes18](https://github.com/Ash-codes18.png?size=60)](https://github.com/Ash-codes18 "HectorSaldes on GitHub")
+[![Maagnitude](https://github.com/Maagnitude.png?size=60)](https://github.com/Maagnitude "Ash-codes18 on GitHub")
+[![cracker911181](https://github.com/cracker911181.png?size=60)](https://github.com/cracker911181 "Maagnitude on GitHub")
+[![quiet-node](https://github.com/quiet-node.png?size=60)](https://github.com/quiet-node "quiet-node on GitHub")
+[![ProfCyberNaught](https://github.com/ProfCyberNaught.png?size=60)](https://github.com/ProfCyberNaught "ProfCyberNaught on GitHub")
+
+Feel free to [open a PR](https://github.com/DenverCoder1/readme-typing-svg/issues/21#issue-870549556 "open a PR") and add yours!
```
Comment (DenverCoder1) [inline README.md:84]: Title text isn't needed when it is the same as the link text

```suggestion
Feel free to [open a PR](https://github.com/DenverCoder1/readme-typing-svg/issues/21#issue-870549556) and add yours!
```

**Review by DenverCoder1 — COMMENTED** [review]
Thanks for working on this, however it seems that most of these instances of added title text are redundant or otherwise intentionally omitted. Note that for SEO purposes, search engines take into account *both* the link text and the title text and there is no reason to add title text if it just duplicating the link text. Additionally, I prefer to not put title text in the example code since it's not functionally required and may confuse some users. Users may choose to add it on their own if they wish.

To summarize, I am fine with adding alt text to the image on line 7 of the readme, and adding title text to the GitHub profile links, but the other instances don't seem to have much meaningful impact on the content or searchability so I think they should be reverted.

Feel free to let me know what you think.

**Comment by ProfCyberNaught** [comment]
This is your repository and indeed your choice moving forward. I don't fully agree with your comments regarding title tags as some search engine do use them, some categorisations stored in databases use them to find keywords, and screen readers would certainly look out for them. However, I do accept if the title tag data is duplicating the element data, there would be no point. However, I certainly was not going to start adding more content and replacing your ideal element descriptions without clearing anything with you first :smile:  I have accepted your reversion changes and do appreciate the time taken to address the concerns and changes made. This should be good to merge having accepted the suggested reversions made above.

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for making this contribution and for the extra effort. It is much appreciated 👍

**Comment by ProfCyberNaught** [comment]
Not a problem. I am glad to help the project out. Was a little concerning collecting the statics regarding the link-backs and a third of users wanting profile links do not even honour you with a link back to the repo or demo site. Very disappointing to have found those statistics :frowning_face:
